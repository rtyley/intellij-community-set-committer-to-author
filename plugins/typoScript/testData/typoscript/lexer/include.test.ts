<INCLUDE_TYPOSCRIPT : source="FILE: fileadmin/html/mainmenu_typoscript.txt">
<INCLUDE_TYPOSCRIPT : source="EXT:myext/directory/file.txt">
<INCLUDE_TYPOSCRIPT : source="FILE: fileadmin/html/mainmenu_typoscript.txt"> t = 8
<
< INCLsome strange text>
< f = 34
< fff.dsfs
