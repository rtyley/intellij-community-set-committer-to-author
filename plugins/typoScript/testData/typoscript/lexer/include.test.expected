INCLUDE ('<INCLUDE_TYPOSCRIPT : source="FILE: fileadmin/html/mainmenu_typoscript.txt">')
WHITE_SPACE_WITH_NEW_LINE ('
')
INCLUDE ('<INCLUDE_TYPOSCRIPT : source="EXT:myext/directory/file.txt">')
WHITE_SPACE_WITH_NEW_LINE ('
')
INCLUDE ('<INCLUDE_TYPOSCRIPT : source="FILE: fileadmin/html/mainmenu_typoscript.txt"> t = 8')
WHITE_SPACE_WITH_NEW_LINE ('
')
INCLUDE ('<')
WHITE_SPACE_WITH_NEW_LINE ('
')
INCLUDE ('< INCLsome strange text>')
WHITE_SPACE_WITH_NEW_LINE ('
')
INCLUDE ('< f = 34')
WHITE_SPACE_WITH_NEW_LINE ('
')
INCLUDE ('< fff.dsfs')
WHITE_SPACE_WITH_NEW_LINE ('
')